### DISCUSSION TITLE
Do I need to update the screen or just the board?

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#137] [Q&A] [answered]
Hello. I've been using my ender3v2 on stock firmware since I got it and had no issues. I'm trying to step outside my comfort zone and I'm excited about the work that's been done with this version of firmware. I have my Manual Mesh 3x3 v4 bin and I get how to flash this. 

What I don't know is do I need to update my screen. If so where would I find the file for that?

**Comment by t3chguy** [comment, marked as answer]
Just the board for now, future updates will add an *optional* screen update to get a wider variety of icons.

**Reply by Tapap84** [reply]
Cool. I appreciate your time. I wasn't sure if I'd need to in order to access features. Have a great day.
